src/test/swift/com/sleekbyte/tailor/functional/CommaWhitespaceTest.swift

```diff
@@ -207,3 +207,49 @@ enum ASCIIControlCharacter: Character {
     case Tab = "\t",LineFeed = "\n"
     case CarriageReturn = "\r"
 }
+
+var (x, y): (Int, Int)
+
+var (x,y): (Int , Int)
+
+var (x, y ,z): (Int,Int, Int)
+
+var (x): Int = 2
+
```

Comment:
that tuple actually gets parsed as a `parenthesized expression`, which I'll implement in a later PR